medusa/server/api/v2/internal.py

```diff
@@ -136,3 +138,92 @@ def resource_existing_series(self):
                 dir_list.append(cur_dir)
 
         return self._ok(data=dir_list)
+
+    # searchIndexersForShowName
+    def resource_search_indexers_for_show_name(self):
+        """
+        Search indexers for show name.
+
+        Query parameters:
+        :param query: Search term
+        :param indexerId: Indexer to search, defined by ID. '0' for all indexers.
+        :param language: 2-letter language code to search the indexer(s) in
+        """
+        query = self.get_argument('query', '').strip()
+        indexer_id = self.get_argument('indexerId', '0')
+        language = self.get_argument('language', '').strip()
+
+        if not query:
+            return self._bad_request('No search term provided.')
+
+        enabled_indexers = indexerApi().indexers
+        indexer_id = try_int(indexer_id)
+        if indexer_id > 0 and indexer_id not in enabled_indexers:
+            return self._bad_request('Invalid indexer id.')
+
+        if not language or language == 'null':
+            language = app.INDEXER_DEFAULT_LANGUAGE
+
+        search_terms = [query]
+
+        # If search term ends with what looks like a year, enclose it in ()
+        matches = re.match(r'^(.+ |)([12][0-9]{3})$', query)
```

Comment:
Can probably add this as another PR but we also need to do `vs` and `vs.` as I found with `Red vs. Blue`.